assets/javascripts/wizard/components/wizard-field-upload.js.es6

```diff
@@ -4,6 +4,7 @@ import { getToken } from "wizard/lib/ajax";
 export default Ember.Component.extend({
   classNames: ["wizard-field-upload"],
   uploading: false,
+  isImage: false,
```

Comment:
Done pending commit

Reply:
@merefield That's true, however it will be useful to target the parent class in certain situations, for example if you want to use flexbox. Adding the classnamebinding gives us a bit more flexibility.

Reply:
Back in pending commit